src/plugins/clearData/test/clearDataActionSpec.js

```diff
@@ -0,0 +1,43 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import ClearDataAction from '../clearDataAction.js';
+
+describe('Clear Data Action', function () {
```

Comment:
Thanks for adding the spec, looks great. I'm actually thinking that we should be defining unit tests at a plugin level. Like, the test could install the plugin, and then check that the action *and* the indicator are installed using spies on `openmct.indicators.add` and `openmct.contextMenu.registerAction`. You could use the `registerAction` spy to get the action created by the plugin and then execute the `emit` check. Thoughts?